ROMFS/px4fmu_common/init.d-posix/1010_iris_opt_flow

```diff
@@ -0,0 +1,30 @@
+#!nsh
```

Comment:
We could use `#!sh` everywhere.
I don't know why we don't currently have a `/bin/` in NuttX.